applets/brightness/brightness-applet.c

```diff
@@ -776,68 +776,56 @@ gpm_applet_stop_scroll_events_cb (GtkWidget *widget, GdkEvent  *event)
 static void
 gpm_applet_dialog_about_cb (GtkAction *action, gpointer data)
 {
-	GtkAboutDialog *about;
-
-	GdkPixbuf *logo =
-		gtk_icon_theme_load_icon (gtk_icon_theme_get_default (),
-					  GPM_BRIGHTNESS_APPLET_ICON,
-					  128, 0, NULL);
-
 	static const gchar *authors[] = {
 		"Benjamin Canou <bookeldor@gmail.com>",
 		"Richard Hughes <richard@hughsie.com>",
 		NULL
 	};
 
-	char copyright[] = \
-		"Copyright \xc2\xa9 2012-2020 MATE developers\n"
-		"Copyright \xc2\xa9 2006 Benjamin Canou";
-
 	const char *documenters [] = {
 		NULL
 	};
+
 	const char *license[] = {
-		 N_("Licensed under the GNU General Public License Version 2"),
```

Comment:
This part of the License text is missing, typo?

Reply:
This line is not part of GPL license notice.

**The license notices** section at [How to use GNU licenses for your own software](https://www.gnu.org/licenses/gpl-howto.en.html).

> Each file's copying permission statement (also called the license notice) should come right after its copyright notices. For a one-file program, the statement (for the GPL) should look like this, to use GPL version 3 or later:

<pre>
    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.
</pre>
...

> To use a different set of GPL versions, you would modify the end of the first long paragraph. For instance, to license under version 2 or later, you would replace “3” with “2”.

...

Reply:
Ok :)